solidity/random-beacon/contracts/libraries/DKG.sol

```diff
@@ -91,16 +91,18 @@ library DKG {
     /// @dev Size of a group in the threshold relay.
     uint256 public constant groupSize = 64;
 
-    /// @dev Minimum number of group members needed to interact according to the
-    ///      protocol to produce a relay entry.
+    /// @dev The minimum number of group members needed to interact according to
+    ///      the protocol to produce a relay entry. The adversary can not learn
```

Comment:
The doc is a little bit confusing as it mentions `relay entry` but is used for DKG result signatures number check.

```
    /// @dev The minimum number of group members needed to interact according to
    ///      the protocol to provide signatures for the DKG result. The adversary can not learn (...)

Reply:
498949806b6f01ec9320d377ce1de1657e592080